client/src/main/java/org/apache/rocketmq/client/latency/LatencyFaultToleranceImpl.java

```diff
@@ -37,11 +37,7 @@ public void updateFaultItem(final String name, final long currentLatency, final
             faultItem.setCurrentLatency(currentLatency);
             faultItem.setStartTimestamp(System.currentTimeMillis() + notAvailableDuration);
 
-            old = this.faultItemTable.putIfAbsent(name, faultItem);
-            if (old != null) {
-                old.setCurrentLatency(currentLatency);
-                old.setStartTimestamp(System.currentTimeMillis() + notAvailableDuration);
-            }
+            this.faultItemTable.putIfAbsent(name, faultItem);
```

Comment:
The original code is used to update the latency to exists fault item. Could you please write a test case to verify your issue?

Reply:
you are right.